news.md

```diff
@@ -4,6 +4,19 @@ permalink: /news.html
 title: JMAP News
 ---
 # JMAP News
+## JMAP Email Specification have been published by Internet Engineering Task Force (IETF), marking a big victory for email tools now and in the future
+
+16 August 2019
+
+>Fastmail engineers have been working for the past five years to bring [JMAP technology](https://fastmail.blog/2014/12/23/jmap-a-better-way-to-email/) to developers and customers. We're proud to announce that [RFC 8620](https://tools.ietf.org/html/rfc8620), published on July 18th, and [RFC 8621](https://tools.ietf.org/html/rfc8621), published on August 13th, have introduced JMAP to the world!
+
+[Continue reading...](https://fastmail.blog/2018/12/27/jmap-is-on-the-home-straight/)
+
+## JMAP Protocol development funding
```

Comment:
I'm not sure what this section is meant to be for; I think it can be cut.

Reply:
I have remove this section

Reply:
The idea was to share this good news: https://github.com/k9mail/k-9/issues/3272#issuecomment-528326161

Reply:
That sounds good if you want to add that in!

Reply:
Should I restore that section?

Reply:
I'm happy to have a section for https://github.com/k9mail/k-9/issues/3272#issuecomment-528326161 – this wasn't what was there before.

Reply:
The section should concern [JMAP protocol issue #3272](https://github.com/k9mail/k-9/issues/3272l) or specificly the comment [k9mail/k-9#3272 (comment)](https://github.com/k9mail/k-9/issues/3272#issuecomment-528326161)